src/pybind/rados/rados.pyx

```diff
@@ -2756,6 +2820,44 @@ returned %d, but should return zero on success." % (self.name, ret))
             raise LogicError("Ioctx.write_full(%s): rados_write_full \
 returned %d, but should return zero on success." % (self.name, ret))
 
+    @requires(('key', str_type), ('data', bytes), ('write_len', int), ('offset', int))
+    def writesame(self, key, data, write_len, offset=0):
+        """
+        Write the same buffer multiple times
+        :param key: name of the object
+        :type key: str
+        :param data: data to write
+        :type data: bytes
+        :param len: total number of bytes to write
```

Comment:
can you change the `len` parameter to `write_length` that  would make it consistent with the `PrimaryLogPG::do_writesame`

Reply:
updated， please check.